voltcraft/pps.py

```diff
@@ -3,6 +3,7 @@
 import sys
 
 import serial
+from typing_extensions import Literal
```

Comment:
Please guard this with


```suggestion
if sys.version_info >= (3, 8):
    from typing import Literal
else:
    from typing_extensions import Literal
```